GDJS/Runtime/pixi-renderers/spriteruntimeobject-pixi-renderer.js

```diff
@@ -131,3 +131,16 @@ gdjs.SpriteRuntimeObjectPixiRenderer.getAnimationFrameWidth = function(pixiTextu
 gdjs.SpriteRuntimeObjectPixiRenderer.getAnimationFrameHeight = function(pixiTexture) {
     return pixiTexture.height;
 };
+
+gdjs.SpriteRuntimeObjectPixiRenderer.setSprite = function(runtimeScene, imageName) {
```

Comment:
The function name is a bit misleading, you would think seeing a `setSprite` method that it accepts a pixi sprite, or a GDevelop sprite. I think something like `changeTexture` or `setTexture` would be more appropriate.